src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -776,6 +831,78 @@ static int Lambda(int x, Func<int, int> l)
 True
 True
 True
+True
+";
+
+            CompileAndVerify(source + InstrumentationHelperSource, expectedOutput: expectedOutput, options: TestOptions.ReleaseExe);
+            CompileAndVerify(source + InstrumentationHelperSource, expectedOutput: expectedOutput, options: TestOptions.DebugExe);
+        }
+
+        [Fact]
+        public void MultipleFilesCoverage()
```

Comment:
Would it make sense to check the generated IL for this method? We should see "Source Document 1/2/3/4", right?

That makes me think: do we need to worry about or gracefully degrade when there are more than Int32.Max (2 billion something) documents?

---
In reply to: [71272776](https://github.com/dotnet/roslyn/pull/12601#discussion_r71272776) [](ancestors = 71272776)